com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolume.hlsl

```diff
@@ -84,18 +84,32 @@ APVConstants LoadAPVConstants( StructuredBuffer<int> index )
 #define APV_USE_BASE_OFFSET
 
 // We split the evaluation in several steps to make variants with different bands easier.
-float3 EvaluateAPVL0(APVResources apvRes, float3 uvw, out float L1Rx)
+float3 EvaluateAPVL0(APVResources apvRes, float3 uvw, out float L1Rx, bool usePointSampling = false)
```

Comment:
This might get compiled to something bad perf wise; I'd write a function just for the debug since there is no situation in runtime in which using point sampling is good. 

This is also because once I do the change with probe position/location in tex, this won't be needed anymore and reverting a self-contained function is easier.

Reply:
Putting comment here but of course referring to the whole file changes

Reply:
Are you sure about that? Last time I checked (which was a long time ago I admit :p ) this worked fine since it was always a compiled time constant.
Regardless, as you said we're going to change that again so I can make a specific function instead.